### ISSUE TITLE
Error on sql template literal : Syntax Error: Unexpected single quote character ('), did you mean to use a double quote (")?ts(52001)

### ISSUE THREAD

**Issue by Hebilicious** [0no-co/GraphQLSP#156]
Hi, trying out this plugin, I'm getting the following error in a file that is unrelated to graphql.
The file is question is some drizzle file that defines postgres tables. 

Using ts-expect-error or ts-ignore doesn't silence the VS code error.

<img width="804" alt="image" src="https://github.com/0no-co/GraphQLSP/assets/13395944/63f7c858-0883-47eb-a0bd-db8e2caaf2df">

The error happens only there, I tried to rename the tag template from `sql` to something like `foo`, without success.

Plugin configuration : 

```json
    "plugins": [
      {
        "name": "@0no-co/graphqlsp",
        "schema": "./graphql/schema.graphql",
        "template": "graphql"
      }
    ]
```

Deps : 

```
    "drizzle-orm": "^0.29.1",
    "typescript": "^5.3.3",
    "@0no-co/graphqlsp": "^0.14.0",
    "graphql": "^16.8.1",
```

**Comment by JoviDeCroock** [comment]
Ah, yes we are missing the strict template check during tagged template finding

**Comment by Hebilicious** [comment]
@JoviDeCroock First time looking at the codebase, but I can try to contribute something ! 

Would something like this work ?  `ts.isTaggedTemplateExpression(node) && IsGraphqlTemplate(node)`

**Comment by JoviDeCroock** [comment]
Yes, we would have to check the tag text and whether it's equal to the configured one

**Comment by Hebilicious** [comment]
<img width="854" alt="image" src="https://github.com/0no-co/GraphQLSP/assets/13395944/8b903b3e-7484-4882-9239-41855208165f">
The error never shows up with tsc though, its just an IDE error.
It seems quite buggy, as vscode reports it multiple time 
<img width="547" alt="image" src="https://github.com/0no-co/GraphQLSP/assets/13395944/4ed1af66-e55d-4708-ada7-1326c1cd8dc5">

**Comment by JoviDeCroock** [comment]
The LSP purposefully does not run during tsc so that makes sense

**Comment by JoviDeCroock** [comment]
Published in 0.14.1

**Comment by Hebilicious** [comment]
@JoviDeCroock Wow thank you, I completely missed this !
